### PR TITLE
Docs/improve install

### PR DESCRIPTION
**Pull request by pauloribeiro-codacy**
I refactored the install.md page to make it more streamlined and easier to follow.

The last part that involves setting the configurations in the values-production.yaml still needs work!

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -2,33 +2,47 @@
...
 * [Install Codacy](install.md)
 
 ## Post-install configuration
 
 After successfully installing Codacy, follow the post-install configuration steps:
 
-*  [Use External Databases](configuration/external-dbs.md)
-*  [Logging](configuration/logging.md)
-*  [Monitoring](configuration/monitoring.md)
+1. Start configuring Codacy in the UI. The onboarding process will guide you through the following steps:
```
Comment (pauloribeiro-codacy) [inline docs/index.md]: What will be the URL to access the Codacy UI?

File: docs/install.md [hunk 2]
```diff
@@ -1,292 +1,68 @@
...
+    codacy-activitiesdb-0                            1/1    Running           0         6m3s
+    codacy-api-6f44c8d48-6bw8z                       1/1    Running           0         6m11s
+    codacy-api-6f44c8d48-h6cl4                       1/1    Running           0         6m11s
+    codacy-api-6f44c8d48-vgbl5                       1/1    Running           0         6m11s
+    codacy-core-786c6f79f-7sn7p                      1/1    Running           0         6m11s
+    codacy-core-786c6f79f-pvg9w                      1/1    Running           0         6m11s
+    [...]
+    ```
+
+1. Use the `values-production.yaml` file as [reference](https://raw.githubusercontent.com/codacy/chart/master/codacy/values-production.yaml):
```
Comment (pauloribeiro-codacy) [inline docs/install.md]: We still need to improve this part, hopefully streamlining it like we did for the previous steps.

File: docs/index.md (hunk 1 above)
Comment (paulopontesm) [reply docs/index.md]: You have to configure the ingress before doing this, maybe thats a better step 1

Comment (pauloribeiro-codacy) [reply docs/index.md]: You're right! I'm going to change it.

File: docs/index.md [hunk 3]
```diff
@@ -2,33 +2,49 @@
...
 * [Install Codacy](install.md)
 
 ## Post-install configuration
 
 After successfully installing Codacy, follow the post-install configuration steps:
 
-*  [Use External Databases](configuration/external-dbs.md)
-*  [Logging](configuration/logging.md)
-*  [Monitoring](configuration/monitoring.md)
+1. Enable the Ingress on `codacy-api`.
```
Comment (pauloribeiro-codacy) [inline docs/index.md]: This needs to be expanded! :sweat_smile:

File: docs/install.md [hunk 4]
```diff
@@ -1,292 +1,64 @@
...
+    codacy-activitiesdb-0                            1/1    Running           0         6m3s
+    codacy-api-6f44c8d48-6bw8z                       1/1    Running           0         6m11s
+    codacy-api-6f44c8d48-h6cl4                       1/1    Running           0         6m11s
+    codacy-api-6f44c8d48-vgbl5                       1/1    Running           0         6m11s
+    codacy-core-786c6f79f-7sn7p                      1/1    Running           0         6m11s
+    codacy-core-786c6f79f-pvg9w                      1/1    Running           0         6m11s
+    [...]
+    ```
+
+1. Modify and use the [reference file](https://raw.githubusercontent.com/codacy/chart/master/codacy/values-production.yaml) `values-production.yaml` to [set up external DBs](configuration/external-dbs.md) (ideally running in a cloud managed Postgres instance).
```
Comment (pauloribeiro-codacy) [inline docs/install.md]: Right now, this part is already becoming simpler to explain. I'll improve it later.

**Pull request merged**